docs/using/tutorial/movingvolume.rst

```diff
@@ -0,0 +1,231 @@
+
+- OS X with `Homebrew <http://brew.sh/>`_ installed.
+- Ubuntu 14.04.
+
+2. Two hosts for two instances of Flocker.
+The options are:
+
+- Two Virtual Machines (VMs) on your local machine.
+  For this tutorial, you are supplied with Vagrant images to create the tutorial environment on VMs using VirtualBox, so you must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_ installed.
+- AWS or Rackspace (you will need an account with root access).
+- Physical hosts with a supported operating system.
+- Any combination of the above.
+
+.. note:: If you choose to use VMs on your local machine, you’ll need at least 4GB RAM.
+
+Contents
+--------
+
+.. contents:: 
+	:local:
+	:backlinks: none
+	:depth: 2
+
+Overview
+^^^^^^^^
+
+You will be controlling your Flocker cluster via the CLI you've installed locally.
+The following diagram illustrates the initial server-side Flocker setup that you will control via the CLI.
+
+.. image:: images/flocker-tutorial-initial-setup.svg
+   :width: 60 %
+   :alt: In the initial server-side Flocker setup there are two servers, one of which has two Docker containers running; one container is a running a web application, the other has a Redis database with a volume.
+   :align: center
+
+The following diagram illustrates how the server-side Flocker setup will be configured at the end of the tutorial:
+
+.. image:: images/flocker-tutorial-final-setup.svg
+   :width: 60 %
+   :alt: Following the completion of this tutorial the server-side Flocker setup will be configured with the web application still running within a container on the first server, while the Redis server with a volume is running on the second server.
+   :align: center
+
+Flocker manages the data migration and the link between the two containers.
+
+.. To find out more about how Flocker manages migration of containers with volumes, see *add link here*
+
+Step 1: Installing the Flocker client
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Flocker client runs locally on your machine, and will control the two instances of Flocker located on the hosts.
+To install the Flocker client, run the following in your terminal:
+
+OS X
+****
+.. task:: test_homebrew flocker-|latest-installable|
+   :prompt: you@laptop:~$
+
+Ubuntu 14.04
+************
+.. task:: install_cli ubuntu-14.04
+   :prompt: you@laptop:~$
+
+To test your installation, run the following to check that you have the Flocker client installed correctly:
+
+.. prompt:: bash [you@laptop:~$]
+
+   flocker-deploy --version
+
+Successful installation will display the version of Flocker.
+
+Step 2: Installing Flocker on your hosts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Option A: Installing Flocker on local VMs
+*****************************************
+
+.. note:: You must have `Vagrant <https://www.vagrantup.com/>`_ and `VirtualBox <https://www.virtualbox.org/>`_  installed to create the VMs and start the containers for this tutorial.
```

Comment:
I don't have too much trouble with the GSG in production. My problem is I was trying to use one in master, which meant I had to install the master CLI via homebrew, amend the Vagrant file, update Vagrant to include the correct master version of Flocker etc etc. So I think it wasn't relevant to this conversation. Anything to reduce the number of steps though will be good in m y view, but messing about too much with it at this stage isn't sensible